website/webpack.config.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 // const autoprefixer = require('autoprefixer');
 // const webpack = require('webpack');
 // const TerserPlugin = require('terser-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
@123MwanjeMike why this this moved here? This webpack is used to build the app only. If you wan to run an index, check the either the `dev-server` or `stand-alone` webpack config.